src/main/groovy/ru/endlesscode/bukkitgradle/meta/MetaFile.groovy

```diff
@@ -64,8 +64,8 @@ class MetaFile {
      */
     private validateMeta() {
         for (MetaItem item in meta.items) {
-            if (!item.valid) {
-                throw new GradleException("Plugin metadata parse error: '$item.id' must not be null")
+            if (!item.validate()) {
+                break
```

Comment:
It is not intuitive that `validate` never returns `false` but throws an exception instead. So `break` will never be reached.

Reply:
Yes, it's stange place, maybe it should be just
```
loop {
  item.validate()
}
```